sound/soc/sof/ipc4-topology.c

```diff
@@ -1490,7 +1477,8 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 	struct snd_soc_component *scomp = swidget->scomp;
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 	struct sof_ipc4_copier_data *copier_data;
-	struct snd_pcm_hw_params ref_params;
+	struct snd_pcm_hw_params *ref_params;
```

Comment:
@ujfalusi this is so confusing. Why do you change this to a local copy in the first only to change it in the follow up?

Reply:
@ranj063, yes, I know, I was taking a different direction then turned back. It is gone in the new version.